core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -482,11 +484,21 @@ oppia.directive('conversationSkin', [
           };
 
           $scope.submitAnswer = function(answer, interactionRulesService) {
-            // For some reason, answers are getting submitted twice when the
-            // submit button is clicked. This guards against that.
+            // Safety check to prevent double submissions from occurring.
+            // It's not clear if this is needed as the original
```

Comment:
I think you can drop this and the next three lines; I meant the safety check to guard against double submissions in general (for future interactions too).